runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -53,393 +58,167 @@ Icons appear at the top of each page in the Runtime Manager documentation, indic
-+
-See xref:sending-data-from-arm-to-external-analytics-software.adoc[Export Data to External Analytics Tools].
-
-
-* *Monitoring*
-+
-The Runtime Manager cloud console includes dashboards that provide performance metrics for all deployed applications.
-+
-With Runtime Manager, you can set up both alerts based on alert conditions on deployed apps and standard notifications to inform you when events occur in your applications.
-You can also create alerts that are triggered by events related to the servers on which they run, such as reaching a certain CPU usage threshold or adding a new node to a cluster.
-+
-See xref:monitoring.adoc[Monitor Applications and Servers].
-
-* *Object store*
-+
-The object store infrastructure is not available for hybrid deployments.
-To use object stores, you must configure a database to store data.
-+
-See xref:connectors::object-store/object-store-connector.adoc[Object Stores].
-
-
-* *Scheduling*
-+
-The scheduling feature is not available for hybrid deployments.
-To schedule tasks, use the Scheduler endpoint element in your flows.
-+
-See xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint].
-
-* *Security updates*
-+
-After an application is deployed and running, you must apply any security updates for the selected runtime version manually.
-
-
-* *Shared resource support*
-+
-See <<Shared Resource Support for On-Premises App Deployments>>.
-
-* *Troubleshooting*
-+
-CloudHub provides the Insight troubleshooting tool that provides in-depth visibility into business transactions and events on your Mule apps deployed through Runtime Manager.
-+
-See xref:insight.adoc[Insight].
-
-
-=== Security for Hybrid Deployments
-
 By default, only metadata pushed by the Runtime Manager agent in each Mule runtime engine flows to the cloud. No application data is exposed.
 
-The agent monitors and controls Mule, and publishes its data to the control plane.
-You can control Mule from external systems by calling Runtime Manager agent APIs, or you can specify that Mule publishes its data to external systems.
+The agent monitors and controls Mule, and publishes its data to the control plane. You can control Mule from external systems by calling Runtime Manager agent APIs, or you can specify that Mule publishes its data to external systems.
 
-You can change the default behavior so that the agent pushes IDs, final average numbers, or any data you find useful for monitoring or keeping control of applications.
+You can change the default behavior so that the agent pushes IDs, final average numbers, or any data you find useful for monitoring or keeping control of applications. 
 See xref:runtime-manager-agent.adoc[Runtime Manager Agent].
 
-You can also tokenize sensitive data.
-See xref:anypoint-security::tokenization-example.adoc[Tokenization Service Components].
-
+You can also tokenize sensitive data. See xref:anypoint-security::tokenization-example.adoc[Tokenization Service Components].
```

Comment:
I think the tokenization business for RTF is being EOLed with the EOL of VM/Bare Metal. Since this is a section on Hybrid, maybe a better link would be:
https://docs.mulesoft.com/anypoint-security/tokenization-from-a-mule